### PR TITLE
FIX: SET isAdmin = 1 WHERE id = 1

### PR DESCRIPTION
**Pull request by sooox-cc**

### CODE REVIEW

File: server.js
Comment (sooox-cc) [inline server.js:1]: - Check for isAdmin column
- Add it if it doesn't exist
- Always check for users and set the first one as admin
- Run the admin check regardless of whether the column was just added or already existed

Comment (SleepingAmi) [reply server.js:1]: > Always check for users and set the first one as admin

Does this *literally* mean always, as in like, redundant checks?

**Review by SleepingAmi — APPROVED** [review]
Works for me

File: server.js
Comment (SleepingAmi) [reply server.js:1]: nvm I'm fkn stupid

**Pull request merged**